storage-proofs/porep/src/stacked/vanilla/proof.rs

```diff
@@ -1161,6 +1161,167 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
 
         Ok((tau, (paux, taux)))
     }
+
+    // Assumes data is all zeros.
+    // Replica path is used to create configs, but is not read.
+    // Instead new zeros are provided (hence the need for replica to be all zeros).
+    fn generate_fake_tree_r_last<TreeArity>(
```

Comment:
Oops, this comment didn't make it: Is it possible to use the 'real' tree_r_last code with a zeroed replica file and have it work as normal?  At a quick glance, this code looks largely duplicated.

Reply:
I see the encoding is dropped in this version.  If it's just that and not beneficial to use a null encoder (or whatever is needed), you can disregard this comment.

Reply:
I started out trying to reuse. Encoding became a pain. And there's no need to indirect through a file since we know the content. Although it might be possible to refactor to share some code, I decided it was fine to duplicate the minimum needed for this specialized path. This also minimizes risk/change for the real and tested code path. I generally agree with your sentiments, though.